docs/learn/documentation/versioned/jobs/samza-configurations.md

```diff
@@ -274,6 +274,8 @@ These properties define Samza's storage mechanism for efficient [stateful stream
 |stores.**_store-name_**.<br>rocksdb.max.log.file.size.bytes|67108864|The maximum size in bytes of the RocksDB LOG file before it is rotated.|
 |stores.**_store-name_**.<br>rocksdb.keep.log.file.num|2|The number of RocksDB LOG files (including rotated LOG.old.* files) to keep.|
 |stores.**_store-name_**.<br>rocksdb.metrics.list|(none)|A list of [RocksDB properties](https://github.com/facebook/rocksdb/blob/master/include/rocksdb/db.h#L409) to expose as metrics (gauges).|
+|stores.**_store-name_**.<br>side.inputs|(none)|For Samza applications with large stores that are periodically populated by a secondary data sources such as HDFS, but otherwise ready-only; Side inputs feature makes it easier and efficient to populate data. Stores configured with side inputs use the the source streams to bootstrap in case of failure thereby, reducing additional copy of the data in changelog. The value is a comma-separated list of streams.<br> Each stream is of the format `system-name.stream-name`. Additionally, applications should add the side inputs to job inputs (`task.inputs`) and configure side input processor (`stores.store-name.side.inputs.processor.factory`).
+|stores.**_store-name_**.<br>side.inputs.processor.factory|(none)|The value is a fully-qualified name of a Java class that implements <a href="../api/javadocs/org/apache/samza/storage/SideInputProcessorFactory.html">SideInputProcessorFactory</a>. It is a required configuration for stores with side inputs (`stores.store-name.side.inputs`).
```

Comment:
nitpick: A SideInputProcessor allows applications to modify or transform incoming side-input-tuples in anyway they desire. 
Samza provides an identity processor, which leaves the input as-is (<Name of the class>?)